pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -273,7 +275,7 @@ buildStdenv.mkDerivation ({
   #   https://bugzilla.mozilla.org/show_bug.cgi?id=1538724
   # elf-hack is broken when using clang+lld:
   #   https://bugzilla.mozilla.org/show_bug.cgi?id=1482204
-  ++ lib.optional ltoSupport "--enable-lto"
+  ++ lib.optional ltoSupport "--enable-lto=cross" # Cross-language LTO.
```

Comment:
For those familiar with LTO (which I'm not), does the comment "Cross-language LTO." add any information, or would they know that already from the code `--enable-lto=cross`?

Reply:
This option is parsed by mozconfig, I think it's not a common phrasing about LTO. At least I've never seen `lto=cross` anywhere else, and I learn its meaning after grepping firefox source tarball.